change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update documentation and use org-level community files in repository",
```

Comment:
Curiosity: How do the community files get updated? Do we have to explicitly do this?

Reply:
Community files are in the special org level .github repo: https://github.com/passageidentity/.github

Reply:
Ha, beat me to it!

Reply:
Cool. I didn't know if there was somehting we had to also do in the repos. Thanks!

Reply:
Nope, that's the beauty of it!!